sound/Note.java

```diff
@@ -7,66 +7,93 @@
-    public final static int HIGH = 120;
-    /** Midi Note Pitch Range */
-    public final static int RANGE = HIGH - LOW;
-    /** Whole-Note Duration (in milliseconds) */
-    public final static int DW = 2400;
-    /** Eighth-Note Duration (in milliseconds) */
-    public final static int DE = DW / 8;
-    /** Quarter-Note Duration */
-    public final static int DQ = DW / 4;
-    /** Half-Note Duration */
-    public final static int DH = DW / 2;
-    /** Midi Minimum Note-On Velocity - Pianissimo */
-    public final static int VPP = 25;
-    /** Midi Maximum Note-On Velocity - Fortissimo */
-    public final static int VFF = 120;
-    /** Midi Note-On Velocity Range */
-    public final static int VRANGE = VFF - VPP;
-    /** Special Note that represents the Null Note */
-    public final static Note NULL_NOTE = new Note(-1,-1,-1,-1);
+    /**Midi-instrument Program Number for PIANO.*/
+    public static final int PIANO = 1;
+    /**Midi-instrument Program Number for ELECPIANO.*/
+    public static final int ELECPIANO  =   4;
+    /**Midi-instrument Program Number for HARPSICHORD.*/
+    public static final int HARPSICHORD =   6;
+    /**Midi-instrument Program Number for VIBES.*/
+    public static final int VIBES       =  11;
+    /**Midi-instrument Program Number for ORGAN.*/
+    public static final int ORGAN       =  19;
+    /**Midi-instrument Program Number for ACCORDIAN.*/
+    public static final int ACCORDIAN   =  23;
+    /**Midi-instrument Program Number for BANJO.*/
+    public static final int BANJO       =  25;
+    /**Midi-instrument Program Number for PIZZICATO.*/
+    public static final int PIZZICATO  =  45;
+    /**Midi-instrument Program Number for VIOLIN.*/
+    public static final int VIOLIN      =  52;
+    /**Midi-instrument Program Number for TRUMPET.*/
+    public static final int TRUMPET     =  56;
+    /**Midi-instrument Program Number for TROMBONE.*/
+    public static final int TROMBONE    =  57;
+    /**Midi-instrument Program Number for VIOLIN2.*/
+    public static final int VIOLIN2     =  59;
+    /**Midi-instrument Program Number for VIOLIN3.*/
+    public static final int VIOLIN3     =  65;
+    /**Midi-instrument Program Number for TENORSAX.*/
+    public static final int TENORSAX    =  66;
+    /**Midi-instrument Program Number for FLUTE.*/
+    public static final int FLUTE       =  73;
+    /**Midi-instrument Program Number for PANFLUTE.*/
+    public static final int PANFLUTE    =  75;
+    /**Midi-instrument Program Number for PIANO1.*/
+    public static final int PIANO1      =  80;
+    /**Midi-instrument Program Number for SYNVOICE.*/
+    public static final int SYNVOICE    =  85;
+    /**Midi-instrument Program Number for BASSDRUM.*/
+    public static final int BASSDRUM    = 116;
+    /**Midi-instrument Program Number for MELODICTOM.*/
+    public static final int MELODICTOM  = 117;
+    /**Midi-instrument Program Number for SNAREDRUM.*/
+    public static final int SNAREDRUM   = 120;
```

Comment:
Symbolic constants that are multiple words should use the underscore character (for example, MELODIC_TOM).  I believe that the names of the constants were derived from the values returned by Java's toString() method invoked on the instrument.  Please check to be sure that these are the correct strings and that an underscore is used whenever the string includes a space.